docs/trace/customizing-the-sdk/README.md

```diff
@@ -488,7 +485,176 @@ shutdown.
 
 ### Guidance for library authors
 
-// TODO: Add details here
+#### TracerProviderBuilder attached registration
```

Comment:
I think this is best to be moved to https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/trace/extending-the-sdk

Reply:
Updated